pkg/build/registry/buildconfig/rest_test.go

```diff
@@ -386,7 +392,9 @@ func TestBuildRESTValidatesUpdate(t *testing.T) {
 				Strategy: api.BuildStrategy{
 					Type: api.STIBuildStrategyType,
 					STIStrategy: &api.STIBuildStrategy{
-						Image: "builder/image",
+						From: kapi.ObjectReference{
```

Comment:
Yeah, that is one of the worries when reading the code. Per yesterday discussion:
- `Image` is a way of specifying external image (eg. docker hub image)
- `From` is a way of specifying image from `ImageRepository`
So you need to validate and use both in all the places with `From` being the preferred one if both are given.
@bparees correct me if I'm wrong